src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -1192,6 +1195,15 @@ def __rec_get_ref(self, ref_list):
                 ret.append(elem)
         return ret
 
+    @classmethod
+    @docval(*get_docval(H5DataIO.__init__))
+    def set_dataio(cls, **kwargs):
+        """
```

Comment:
Can you clarify where this function is being used? Also the name implies that dataio is being set but the docstring implies that it is a generator method to create an H5DataIO object?

Reply:
I use it some other code. It's mainly for convenience so you don't have to import H5DataIO. I can rename it or add more clarity to the docstring if you prefer